dialog.js

```diff
@@ -204,7 +204,11 @@ async function setCertDisplayFields(result) {
       document.getElementById('certimg').innerHTML = "✅";
     }
   } else {
-    document.getElementById('certimg').innerHTML = "❌";
+    if (warning == null && location != null) {
```

Comment:
(nit) In JS, prefer `===` over `==` and `!==` over `!=` in (almost?) all cases.

Reply:
Done.